x/gamm/client/cli/tx.go

```diff
@@ -290,6 +291,17 @@ Ex) 2,4,1,5 -> [(Balancer 2, CL 4), (Balancer 1, CL 5)]
 	return cmd
 }
 
+func NewMigrateSharesToFullRangeConcentratedPositionCmd() *cobra.Command {
+	cmd := osmocli.TxCliDesc{
+		Use:              "migrate-shares-to-full-range-concentrated-position [shares-to-migrate]",
+		Short:            "migrate shares to full range concentrated position",
```

Comment:
hey @phamminh0811, can you refer to this and then update `Short` above: https://github.com/osmosis-labs/osmosis/blob/02afa8fac567502849e745abbb2040078ef287ff/x/gamm/pool-models/balancer/codec.go#L20